puppet/zulip/manifests/postgresql_base.pp

```diff
@@ -1,38 +1,38 @@
 # Minimal shared configuration needed to run a Zulip postgres database.
-class zulip::postgres_appdb_base {
-  include zulip::postgres_common
+class zulip::postgresql_base {
+  include zulip::postgresql_common
```

Comment:
Thanks for changing this.  The background for the name `postgres_appdb` was that in the early Zulip days in ~2013, there were other postgres instances being managed for tools like Trac and mediawiki.